Tech@HubAPI/Tech@HubAPITest/GitServiceTest.cs

```diff
@@ -58,6 +58,29 @@ public void TestGetBranchesNotExisting()
             run.Should().Throw<DirectoryNotFoundException>();
         }
 
+        [Fact]
+        public void TestGetTags()
+        {
+            _fileSystem.ImportFolder("./SampleGitRepos/testTags.git", "git/testUser/testTags.git");
+
+            List<Tag> tags = _gitService.GetTags("testUser", "testBranches");
```

Comment:
I think the test is failing because the repository you "created" is named `testTags` but here you search for `testBranches`